spec/helpers/any.js

```diff
@@ -44,6 +44,27 @@ describe('any helper (with option hash)', function() {
                 input: '{{#any arrayWithObjs num=2}}{{big}}{{/any}}',
                 output: '',
             },
+            {
+                input: '{{#any foobar num=2}}{{big}}{{/any}}',
+                output: '',
+                context: {
+                    foobar: null
+                }
+            },
+            {
+                input: '{{#any foobar num=2}}{{big}}{{/any}}',
+                output: '',
+                context: {
+                    foobar: undefined
+                }
+            },
+            {
+                input: '{{#any undefined num=1}}{{big}}{{/any}}',
```

Comment:
If you're trying to test for `undefined`, you probably want `{{#any num=1}}{{big}}{{/any}}`

Reply:
Will add this in #105